src/main/webapp/app/shared/pipes/filterfeedbacks.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { PipeTransform, Pipe } from '@angular/core';
+import { Feedback } from 'app/entities/feedback';
+
+@Pipe({ name: 'filterfeedbacks' })
+export class FilterFeedbacksPipe implements PipeTransform {
+    transform(feedbacks: Feedback[]): any {
+        if (!feedbacks) {
+            return feedbacks;
+        }
+        return feedbacks.filter(feedback => !(feedback.type === 'AUTOMATIC' && feedback.positive));
```

Comment:
I'm not as deep in this topic as you, maybe a comment would be good, why we remove only automatic feedback.

Reply:
I've added a comment, I hope its more clear now.